2023/did-wg.html

```diff
@@ -337,13 +315,18 @@ <h2>Success Criteria</h2>
 more implementations interoperating with each other. In order to advance to
 Proposed Recommendation, each normative specification must have an open
 test suite of every feature defined in the specification.</p>
+
+    <p>In order for <a href="#deliverable-did-resolution">DID Resolution</a> to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, it is expected to define a dummy DID method meant only for demonstrating interoperability across independent implementations. While required for all complying implementations, this method will be explicitly described as unfit for use in production.
+    It is also expected that the Working Group can exhibit that at least two independant open specification of DID methods relying on <a href="#deliverable-did-core">DID Core</a> and <a href="#deliverable-did-resolution">DID Resolution</a>.
```

Comment:
Isn't this required for all TR track documents? 

Specifically all normative statements need to be tested by 2 independent implementations.

I don't see why this sentence is needed, and it could be misread as "the working group will endorse 2 specific did methods (maintained outside the wg) that are suitable for production"... thats twice the problem of standardizing a specific did method.

Reply:
> Isn't this required for all TR track documents?
> 

Not necessarily. The requirements for guidelines (like WCAG) or vocabularies may be different. Hence, the requirement is that a charter must explicitly define its minimal CR requirement.

Reply:
> it could be misread as "the working group will endorse 2 specific did methods (maintained outside the wg) that are suitable for production"

I don't think there is any assumption that a WG endorse the implementations its mentions in its implementation report. Similarly, this risk of misreading seems a bit remote. However, feel free to propose a better wording if you think it can be made clearer.

Reply:
I proposed a new wording of this requirement in 1ecaa35.
@OR13 does it address your concern? If so, are we good to close this conversation?

Reply:
Resolving, as the conversation is being continued [here](https://github.com/w3c/charter-drafts/pull/448#discussion_r1350767221)